.github/workflows/stackhpc-all-in-one.yml

```
@@ -70,17 +70,35 @@ jobs:
   all-in-one:
     name: All in one
     if: inputs.if
-    runs-on: [self-hosted, stackhpc-kayobe-config-aio]
+    runs-on: arc-skc-aio-runner
     permissions: {}
     env:
       KAYOBE_ENVIRONMENT: ci-aio
       KAYOBE_VAULT_PASSWORD: ${{ secrets.KAYOBE_VAULT_PASSWORD }}
       KAYOBE_IMAGE: ${{ inputs.kayobe_image }}
     steps:
+      - name: Upgrade apt
+        run: sudo apt-get update
+
+      - name: Install Package
+        uses: ConorMacBride/install-package@main
+        with:
+          apt: build-essential git unzip nodejs
+
+      - name: Upgrade python version
+        run: sudo apt-get -y install python3 pip python-is-python3
+
+      - name: Install wheel from pip
+        run: sudo pip install wheel        
+      
```

Let's tidy this up a bit
```suggestion
      - name: Install Package
        uses: ConorMacBride/install-package@main
        with:
          apt: build-essential git unzip nodejs python3 python3-pip python3-wheel python-is-python3

```

---

I have previously tried this, however, I had some issues with installing some of the packages using the 'install-package main'. However, maybe it was because of a previous error.